recipe/meta.yaml

```diff
@@ -22,6 +22,7 @@ requirements:
     - gnuconfig  # [unix]
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
+    - sysroot_linux-64 2.17  # [linux64]
```

Comment:
I'm  seeing a new linking  error for PDAL/GDAL which I think is a result of this change
```
/lidar/miniforge3/envs/pdal/bin/../lib/gcc/x86_64-conda-linux-gnu/9.4.0/../../../../x86_64-conda-linux-gnu/bin/ld: /lidar/miniforge3/envs/pdal/lib/libgdal.so.30: undefined reference to `posix_spawnp@GLIBC_2.15'
/lidar/miniforge3/envs/pdal/bin/../lib/gcc/x86_64-conda-linux-gnu/9.4.0/../../../../x86_64-conda-linux-gnu/bin/ld: /lidar/miniforge3/envs/pdal/lib/libgdal.so.30: undefined reference to `memcpy@GLIBC_2.14'
```

Reply:
Hmm, I was kind of afraid of something like that. Can you open an issue with reproduction steps?

Reply:
Resolved with 

```
conda install -c conda-forge sysroot_linux-64==2.17
```

This probably means that everyone depending on GDAL is going to need an update and this should have been a migration, right?

Reply:
Hmm, I though only things listed in https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/master/recipe/conda_build_config.yaml need migration, and `sysroot_linux-64 is` not pinned there. If we were to set up a migration what would we be migrating?

Reply:
I think the PDAL feedstock would need to be rebuilt. Anything else that linked against libgdal.so on linux is going to need to be recompiled too, right?

Reply:
Yes that makes sense. Sorry for mucking things up!

Reply:
@isuruf or @ocefpaf, can you provide some guidance? Do we need to migrate GDAL's dependencies due to this update?

Reply:
Do you have a build log?

Reply:
The `gdal` build log is at https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=457917&view=logs&jobId=d0d954b5-f111-5dc4-4d76-03b6c9d0cf7e&j=d0d954b5-f111-5dc4-4d76-03b6c9d0cf7e&t=841356e0-85bb-57d8-dbbc-852e683d1642

Reply:
No, for pdal.

Reply:
Sorry, I don't have one. It was a manual build that I started with 

```
conda install -c conda-forge compilers ninja cmake -y
conda install -c conda-forge pdal --only-deps -y
```

Reply:
Can you retry and send a verbose log?

Reply:
I tried rebuilding `pdal` and uploaded the log to https://www.dropbox.com/s/wekn0qtvhpht7aj/pdal_linux_64_openssl1.1.1.log?dl=0

Reply:
Can you make it verbose? `make VERBOSE=1`

Reply:
here's the first linking failure with verbose in in my log
```
bin/pdal_io_sbet_reader_test
: && /lidar/miniforge3/envs/pdal2/bin/x86_64-conda-linux-gnu-c++ -fvisibility-inlines-hidden -std=c++17 -fmessage-length=0 -march=nocona -mtune=haswell -ftree-vectorize -fPIC -fstack-protector-strong -fno-plt -O2 -ffunction-sections -pipe -isystem /lidar/miniforge3/envs/pdal/include -O3 -DNDEBUG -Wl,-O2 -Wl,--sort-common -Wl,--as-needed -Wl,-z,relro -Wl,-z,now -Wl,--disable-new-dtags -Wl,--gc-sections -Wl,-rpath,/lidar/miniforge3/envs/pdal/lib -Wl,-rpath-link,/lidar/miniforge3/envs/pdal/lib -L/lidar/miniforge3/envs/pdal/lib -Wl,-rpath-link,/lidar/miniforge3/envs/pdal2/lib test/unit/CMakeFiles/pdal_test_support.dir/Support.cpp.o test/unit/CMakeFiles/pdal_io_sbet_reader_test.dir/io/SbetReaderTest.cpp.o -o bin/pdal_io_sbet_reader_test  -Wl,-rpath,/home/ubuntu/PDAL/build/lib  lib/libpdal_base.so.13.0.0  lib/libpdal_util.so.13.0.0  lib/libgtest.a  -lpthread && :
/lidar/miniforge3/envs/pdal2/bin/../lib/gcc/x86_64-conda-linux-gnu/9.4.0/../../../../x86_64-conda-linux-gnu/bin/ld: /lidar/miniforge3/envs/pdal/lib/libgdal.so.30: undefined reference to `posix_spawnp@GLIBC_2.15'
/lidar/miniforge3/envs/pdal2/bin/../lib/gcc/x86_64-conda-linux-gnu/9.4.0/../../../../x86_64-conda-linux-gnu/bin/ld: /lidar/miniforge3/envs/pdal/lib/libgdal.so.30: undefined reference to `memcpy@GLIBC_2.14'
collect2: error: ld returned 1 exit status
```

Reply:
https://www.dropbox.com/s/54tziq72tl5zyyh/pdal_verbose_linux_64_openssl1.1.1.log?dl=0

Reply:
Ah, that's an issue. Can we revert this PR for a bit and open an issue in conda-forge.github.io?

Reply:
Can you check if adding `export LDFLAGS="$LDFLAGS -Wl,--allow-shlib-undefined"` fixes it?

Reply:
Yep, I will prepare a PR to revert this. @hobu will you open the issue?

Reply:
Oh, and I'm sorry for the hassle everyone, thanks for helping sort it out.

Reply:
Yes, it does work with `export LDFLAGS="$LDFLAGS -Wl,--allow-shlib-undefined"`, full log at https://www.dropbox.com/s/ziqef5ew8mf4s70/pdal_ldflag_verbose_linux_64_openssl1.1.1?dl=0

Should I still revert this?

Reply:
Can you send a PR to add the flag to https://github.com/conda-forge/ctng-compiler-activation-feedstock/blob/master/recipe/conda_build_config.yaml#L87-L96 ?

Reply:
Done in https://github.com/conda-forge/ctng-compiler-activation-feedstock/pull/62

Reply:
@hobu I think this has been addressed by https://github.com/conda-forge/ctng-compiler-activation-feedstock/pull/62 can you check and see if it works now?

Reply:
Confirmed that this cured the issue we were seeing in PDAL. Thanks!